go.mod

```diff
@@ -0,0 +1,63 @@
+module github.com/netlify/gocommerce
+
+require (
```

Comment:
Did you have to mess around with the import strings much by hand?  I had some issues last time I tried to migrate over.